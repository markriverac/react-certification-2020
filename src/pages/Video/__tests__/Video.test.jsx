import React from 'react';
import renderer from 'react-test-renderer';
import Enzyme from 'enzyme';
import { BrowserRouter } from 'react-router-dom';
import Adapter from 'enzyme-adapter-react-16';
import VideoPage from '../Video.page';

Enzyme.configure({ adapter: new Adapter() });

it('Renders without exploding', () => {
  const notFound = renderer.create(
    <BrowserRouter>
      <VideoPage />
    </BrowserRouter>
  );
  const tree = notFound.toJSON();
  expect(tree).toMatchSnapshot();
});
