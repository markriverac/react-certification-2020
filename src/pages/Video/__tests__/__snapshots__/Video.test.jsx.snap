// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders without exploding 1`] = `
<div
  className="wrapper container"
>
  <div
    className="list-container"
  />
</div>
`;
